addons/mail/views/mail_templates.xml

```diff
@@ -44,6 +44,12 @@
             </xpath>
         </template>
 
+        <template id="assets_common" name="attachment popup assets" inherit_id="web.assets_common">
```

Comment:
Due to 'website live chat', there is use thread.js for live chat widget and we create an object ('document_viewer') into the thread widget. that's why we put into the common assets otherwise we need to use both the assets (front end assets and back end assets) and runbot also red if not put into the frontend assets.